Sources/StreamChatUI/ChatMessageList/RETHINK/MessageListVC.swift

```diff
@@ -287,6 +286,15 @@ class MessageListVC<ExtraData: ExtraDataTypes>: _ViewController, UICollectionVie
         collectionView.scrollToItem(at: IndexPath(item: 0, section: 0), at: .bottom, animated: animated)
     }
 
+    @objc
+    private func didLongPress(_ gesture: UILongPressGestureRecognizer) {
+        let location = gesture.location(in: collectionView)
+
+        guard gesture.state == .began else { return }
+        guard let ip = collectionView.indexPathForItem(at: location) else { return }
```

Comment:
nit: why not join those 2 guards together?

Reply:
that's good point, joined ✅